tsconfig.json

```diff
@@ -3,11 +3,9 @@
     "compilerOptions": {
         "sourceMap": false,
         "moduleResolution": "node",
-        "outDir": "dist/types",
-        "target": "es6",
```

Comment:
Why did we remove this option?

Reply:
This setting is in the basic configuration